src/errors.js

```diff
@@ -35,4 +36,21 @@ const exceptions = _.reduce(
   {}
 );
 
-module.exports = exceptions;
+const isRequireError = ({ name, message }) =>
+  name === 'ReferenceError' && message === 'require is not defined';
+
+const handleError = err => {
+  const { RequireModuleError } = exceptions;
+  if (isRequireError(err)) {
+    throw new RequireModuleError(
+      "Node's module system is not in scope. Use z.require() instead."
```

Comment:
I might change the message to be more clear. Remember, we're targeting less technical users. 

* probably use `instead of "require()"` to help them self-service
* I'd change the start to be `For technical reasons, use z.require(...`, since the message about scope doesn't add much

Also if we were real fancy and could easily grab the name of the module they were trying to import, we could do:

`Node's module system is not in scope. Use z.require("lodash") instead of require("lodash").`